src/components/inputs/ColorSelect2/ColorSelect2.test.tsx

```diff
@@ -232,4 +234,24 @@ describe('ColorSelect2', () => {
 
     expect(renderedLabel.innerHTML).toBe(label);
   });
+
+  it('Closes the color picker on outside click', async () => {
+    const mockFn = jest.fn();
```

Comment:
Please, add one more test, where the button has a callback calling `stopPropagation()`. To cover this fix.

Reply:
Done.

Reply:
@udidol please, mark as resolve